Gemfile

```diff
@@ -2,7 +2,7 @@ source 'https://rubygems.org'
 
 ruby '2.3.1'
 
-gem 'rails', '~> 4.2.7.1'
+gem 'rails', '~> 5.0', '>= 5.0.0.1'
```

Comment:
I'm not familiar with passing two version constraints. Shouldn't `~> 5.0`  be enough? I thought it would also match `5.0.0.1`